<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ausbilder</class>
 <widget class="QDialog" name="Ausbilder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>889</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>970</width>
    <height>889</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Ausbilder</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>949</width>
       <height>181</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Ausbilder</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <widget class="QLabel" name="l_Familienname">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Familienname:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_Familienname">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>231</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_Telefon">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="l_Telefon">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Telefon-Nr.:</string>
      </property>
     </widget>
     <widget class="QLabel" name="l_Eintrittsdatum">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Eintrittsdatum:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_Vorname">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>30</y>
        <width>231</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="l_Vorname">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>30</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Vorname:</string>
      </property>
     </widget>
     <widget class="QLabel" name="l_Austrittsdatum">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>130</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Austrittsdatum:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sb_Wochenstunden">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>80</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>50</number>
      </property>
     </widget>
     <widget class="QLabel" name="l_Wochenstunden">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>80</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Wochenstunden:</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="de_Eintritt">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>130</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QDateEdit" name="de_Austritt">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>130</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2090</year>
        <month>12</month>
        <day>31</day>
       </datetime>
      </property>
      <property name="maximumDateTime">
       <datetime>
        <hour>23</hour>
        <minute>59</minute>
        <second>59</second>
        <year>2090</year>
        <month>12</month>
        <day>31</day>
       </datetime>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>949</width>
       <height>71</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Wochentage</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <widget class="QCheckBox" name="cb_montag">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Montag</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_Dienstag">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>30</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Dienstag</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_Mittwoch">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>30</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Mittwoch</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_Donnerstag">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>30</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Donnerstag</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_Freitag">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>30</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Freitag</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>949</width>
       <height>30</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="QPushButton" name="pb_suchen">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Suchen</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../_Symbole/UsedSymbols/searchmagnifierinterfacesymbol1_79893.ico</normaloff>../../_Symbole/UsedSymbols/searchmagnifierinterfacesymbol1_79893.ico</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_leeren">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Leeren</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../_Symbole/UsedSymbols/-clear_90704.ico</normaloff>../../_Symbole/UsedSymbols/-clear_90704.ico</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_Speichern">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Speichern</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../_Symbole/UsedSymbols/save_3621.ico</normaloff>../../_Symbole/UsedSymbols/save_3621.ico</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_Loeschen">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Löschen</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../_Symbole/UsedSymbols/seo-social-web-network-internet_262_icon-icons.com_61518.ico</normaloff>../../_Symbole/UsedSymbols/seo-social-web-network-internet_262_icon-icons.com_61518.ico</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_Abbrechen">
          <property name="minimumSize">
           <size>
            <width>118</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Hauptmenü</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../_Symbole/UsedSymbols/log-out_icon-icons.com_50106.ico</normaloff>../../_Symbole/UsedSymbols/log-out_icon-icons.com_50106.ico</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_excel">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Excel</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../_Symbole/UsedSymbols/microsoft_excel_icon_132212.ico</normaloff>../../_Symbole/UsedSymbols/microsoft_excel_icon_132212.ico</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_report">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Report</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../_Symbole/UsedSymbols/reporting_noun_report_document_file_icon_148360.ico</normaloff>../../_Symbole/UsedSymbols/reporting_noun_report_document_file_icon_148360.ico</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_Hilfe">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Hilfe</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../_Symbole/UsedSymbols/info_13213.ico</normaloff>../../_Symbole/UsedSymbols/info_13213.ico</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tw_anzeige">
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lab_INFO">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>le_Familienname</tabstop>
  <tabstop>le_Vorname</tabstop>
  <tabstop>le_Telefon</tabstop>
  <tabstop>sb_Wochenstunden</tabstop>
  <tabstop>de_Eintritt</tabstop>
  <tabstop>de_Austritt</tabstop>
  <tabstop>cb_montag</tabstop>
  <tabstop>cb_Dienstag</tabstop>
  <tabstop>cb_Mittwoch</tabstop>
  <tabstop>cb_Donnerstag</tabstop>
  <tabstop>cb_Freitag</tabstop>
  <tabstop>pb_suchen</tabstop>
  <tabstop>pb_Speichern</tabstop>
  <tabstop>pb_Loeschen</tabstop>
  <tabstop>pb_excel</tabstop>
  <tabstop>pb_report</tabstop>
  <tabstop>pb_Hilfe</tabstop>
  <tabstop>tw_anzeige</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
